src/shapes/Annotation.js

```diff
@@ -451,7 +451,7 @@ define([
                 this.pickColor = dc.uniquePickColor();
             }
 
-            program.loadOpacity(gl, this.attributes.opacity);
+            program.loadOpacity(gl, this.attributes.opacity * dc.currentLayer.opacity);
```

Comment:
Would it be possible to deprecate AnnotationAttributes.opacity? It seems redundant to have an alpha channel on the colors and a separate opacity attribute.